src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState_Checksum.cs

```diff
@@ -125,16 +125,17 @@ public async Task<Checksum> GetChecksumAsync(ProjectId projectId, CancellationTo
                 }
 
                 ChecksumCollection? frozenSourceGeneratedDocumentIdentities = null;
-                ChecksumsAndIds<DocumentId>? frozenSourceGeneratedDocuments = null;
+                ChecksumsAndIds<DocumentId>? frozenSourceGeneratedDocumentTexts = null;
```

Comment:
this got simplified to avoid an extra indirection and roundtrip.  previously we would store the DocumentStateCheckums for the frozen SG docs.  Then, OOP would ask for these, just to get the text checksum, just to call back into the host to get this.  This changes to now just store the text checksums only.